terms-of-service.md

```diff
@@ -0,0 +1,213 @@
+
+# TERMS OF SERVICE OF USERSTYLES.WORLD
+Last updated May 5, 2021
+
+
+## AGREEMENT TO TERMS
+
+These Terms of Service constitute a legally binding agreement made between you, whether personally or on behalf of an entity (“you”) and Userstyles world (“we,” “us” or “our”), concerning your access to and use of the userstyles.world website as well as any other media form, media channel, mobile website or mobile application related, linked, or otherwise connected thereto (collectively, the “Site”). 
```

Comment:
```suggestion
These Terms of Service constitute a legally binding agreement made between you, whether personally or on behalf of an entity (“you”) and UserStyles.world (“we,” “us” or “our”), concerning your access to and use of the userstyles.world website as well as any other media form, media channel, mobile website or mobile application related, linked, or otherwise connected thereto (collectively, the “Site”). 
```